chainer/functions/connection/embed_id.py

```diff
@@ -6,6 +6,7 @@
 from chainer.backends import cuda
 from chainer import function_node
 from chainer.utils import type_check
+from chainer.utils import non_deterministic
```

Comment:
"Do not import objects, only modules".
https://docs.openstack.org/hacking/latest/user/hacking.html#imports